README.md

```diff
@@ -31,13 +31,13 @@ Replace `<version>` with a specific version for stable builds. Omitting `<:versi
 docker pull bhargavsnv/cryptgo:<version>
 
 # Run image
-docker run --rm -it -v "$HOME:/home/appuser" bhargavsnv/cryptgo
+docker run -u $(id -u ${USER}):$(id -g ${USER}) -v "$HOME:/home/appuser/" --rm -it bhargavsnv/cryptgo
 ```
 
 Optionally, an alias can be created for ease of use:
 
 ```bash
-alias cryptgo='docker run --rm -it -v "$HOME:/home/appuser" bhargavsnv/cryptgo'
+alias cryptgo='docker run -u $(id -u ${USER}):$(id -g ${USER}) -v "$HOME:/home/appuser/" --rm -it bhargavsnv/cryptgo'
```

Comment:
Any reason as to why you're using `$(id -u ${USER})` instead of maybe just running `$(id -u)`?

Reply:
No reason actually, just to be sure. But the simpler one should work aswell, will update in a bit.